contrib/remote-dev/README.md

```diff
@@ -0,0 +1,183 @@
+# Remote Development
+
+## 1. Overview
+
+`Remote Development` can allow users to develop locally with PAI's resources and improve users’ development experience.
+
+The main process of `Remote Development`:
+- Ask PAI for a container with resources using `start.py`
+- Get SSH info
+- Configure local editors or IDEs
+- Develop and debug in this container like local
+
+Support IDEs or editors:
+- CLI
+- Visual Studio Code
+- PyCharm
+
+Support OS:
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Windows 10
+
+***NOTICE: This is still an experiment solution and may be changed in a future release.***
+
+## 2. Directory Structure
+
+```
+remote-dev/
+├── README.md
+├── conf
+│   ├── clusters.template        # PAI cluster template conf
+│   └── job.template             # PAI template job
+└── start.py                     # start script
+```
+
+## 3. Dependencies
+
+- This subproject depends on [PAI's Python SDK](https://github.com/microsoft/pai/tree/master/contrib/python-sdk), please install and test the SDK first.
+
+- The `start.py` script requires python3, and we only tested it on `py3.5+` environment.
```

Comment:
could you make it compatible with both py2 and py3?

Reply:
Nope, because python-sdk only support py3.

Reply:
...